test/establishments/establishment.test.js

```diff
@@ -970,6 +972,89 @@ describe ("establishment", async () => {
                 .expect(200);
             expect(namedServicePostResponse.body.serviceUsers[0].id).toEqual(21);
         
+            // Block of four "other" field tests
+
+            // should update service users with other but have no consequence if other is passed to a 'non other' service
+            const otherOnNonOtherPostResponse = await apiEndpoint.post(`/establishment/${establishmentUid}/serviceUsers`)
+                .set('Authorization', authToken)
+                .send({
+                    serviceUsers: [
+                        {
+                            service: 'Carers of adults',
+                            other: "my example other"
+                        }
+                    ]
+                })
+                .expect('Content-Type', /json/)
+                .expect(200);
+            expect(otherOnNonOtherPostResponse.body.serviceUsers[0].id).toEqual(21);
+        
+            changeHistory =  await apiEndpoint.get(`/establishment/${establishmentUid}/serviceUsers?history=property`)
+            .set('Authorization', authToken)
+            .expect('Content-Type', /json/)
+            .expect(200);
+
+            chai.expect(changeHistory.body.serviceUsers.currentValue).to.eql([{ id: 21,
```

Comment:
jest has a comprehensive `expect` set, that means we don't need chai. It is necessary here to simply:

`expect(otherOnNonOtherPostResponse.body.serviceUsers[0]).not.toHaveProperty('other');`